### DISCUSSION TITLE
TabView from Xamarin.Community.Toolkit

### DISCUSSION THREAD

**Discussion by bluedrakim** [CommunityToolkit/Maui#1253] [New Feature Discussions]
When I was using Xamarin, I used TabView from Xamarin.Community.Toolkit a lot because it was so easy to create tabs. My struggle with .NET MAUI is to create inner tabs from outer tabs. I found this article about top and bottom bars but my case is different because this article shows if user clicks inner tabs, the entire screen changes to that tab. What I would love to see is that I can set the location of the tabs and only the area where I allow shows the content of the tab and the rest are not affected by the tabs. It was possible to do this with TabView as for instance, I can create an outer tab, and in that tab, I can create 3 tabs that takes the bottom half of the screen. so while I'm switching tabs from those 3 tabs, only the bottom half of the screen changes and the top half of the screen is not affected by the switch.
https://learn.microsoft.com/en-us/dotnet/maui/fundamentals/shell/tabs#bottom-and-top-tabs

**Comment by VladislavAntonyuk** [comment]
track it here; https://github.com/CommunityToolkit/Maui/issues/121
